### PR TITLE
move bundle task over to gulpfile

### PR DESCRIPTION
**Pull request by meandavejustice**
Moves bundle/zip task to gulpfile for consistency

### CODE REVIEW

**Comment by 6a68** [comment]
@meandavejustice looks like this pup needs a rebase and a `gulp lint` pass

File: gulpfile.js [hunk 1]
```diff
@@ -30,10 +31,22 @@ gulp.task('selfie', function(){
...
+gulp.task('bundle', function() {
+  var mainFilter = filter(['**/*', '.**/*',
+                           '!.git', '!.git/**',
+                           '!src', '!src/**',
+                           '!node_modules', '!node_modules/**']);
+
+  return gulp.src(['**/*', '.**/*'])
+         .pipe(mainFilter)
+         .pipe(zip('app-'+ new Date().toString() + '.zip'))
+         .pipe(gulp.dest('../'))
```
Comment (6a68) [inline gulpfile.js]: This seems to put the zip file outside the repo. Could we instead create an output directory inside this repo, put the zipfile there, and add the directory to .gitignore?

File: README.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Don't forget to set your AWS creds in `.ebenvironment/environment.config`
 point your browser to localhost:3001
 
 ## Deploy
-You will need to run `npm run bundle` to create the zip and upload it through the
+You will need to run `gulp bundle` to create the zip and upload it through the
```
Comment (6a68) [inline README.md]: Would you mind adding a comment to explain where the zip file is created?

File: gulpfile.js (hunk 1 above)
Comment (pdehaan) [inline gulpfile.js]: +1 to not polluting my parent directory and just adding some .gitignored output file/folder to ignore the bundle zip.

**Comment by 6a68** [comment]
Nice work, @meandavejustice! Couple of minor things above, overall looks great.

File: gulpfile.js [hunk 3]
```diff
@@ -30,10 +31,22 @@ gulp.task('selfie', function(){
...
+})
```
Comment (pdehaan) [inline gulpfile.js]: Missing semis (x2):

```
[22:54:37] Using gulpfile ~/build/mozilla/idea-town/gulpfile.js
[22:54:37] Starting 'lint'...
[22:54:37] 'lint' errored after 206 ms
[22:54:37] ESLintError in plugin 'gulp-eslint'
Message:
    Missing semicolon.
Details:
    fileName: /home/travis/build/mozilla/idea-town/gulpfile.js
    lineNumber: 43
[22:54:37] 
/home/travis/build/mozilla/idea-town/gulpfile.js
  43:33  error  Missing semicolon  semi
  44:3   error  Missing semicolon  semi
✖ 2 problems (2 errors, 0 warnings)
```

**Comment by 6a68** [comment]
Works for me, merging :beers:

**Pull request merged**